applications/blink_raw/targets/freertos.mips4k.pic32mx.lccbuffer/nbproject/Makefile-default.mk

```diff
@@ -0,0 +1,28 @@
+#
```

Comment:
I have installed MPLAb on my linux machine and have managed to produce the errors on being unable to load the elf file. Let me see if I can do something with that.